src/main/app/case/case.ts

```diff
@@ -590,8 +590,8 @@ export interface Case {
   draftOrderDoc?: Document;
   withdrawApplication?: YesOrNo;
   withdrawApplicationReason?: string;
-  isCafcassServed?: YesOrNo | null;
-  isCafcassCymruServed?: YesOrNo | null;
+  soaCafcassServedOptions?: YesOrNo | null;
```

Comment:
Please check with @sealDebolina on the implementation. Looks like things have changed wrt backend prop